db/migrate/20160712095859_make_existing_foreign_keys_polymorphic_for_comments.rb

```diff
@@ -0,0 +1,35 @@
+class Comment < ActiveRecord::Base
+  belongs_to :team
+  belongs_to :application
+  belongs_to :project
+end
+
+class MakeExistingForeignKeysPolymorphicForComments < ActiveRecord::Migration
+  def up
+    ActiveRecord::Base.transaction do
+      Comment.includes(:team, :application, :project).find_each do |comment|
+        if assoc = [:team, :application, :project].detect { |a| comment.send(a).present? }
```

Comment:
*(accidentally deleted my comment again I guess..)*
Awesome approach using `detect` here I wanted to say 🎉